### PR TITLE
Feature fix brett

### PR DESCRIPTION
**Pull request by jared-h-wood**
This is the authoritative version of this code

### CODE REVIEW

File: Brett/BrettFile.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-Brett is Awesomer than Jared!!!!
-
-Truth is hard, I'm sorry.
\ No newline at end of file
+Brett is a husband and father.
```
Comment (jrobber) [inline Brett/BrettFile.txt:1]: You are so nice

File: Jared/lies.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
-Something about pandas.
+This file has 7 lines in it
+2 + 4 = 25
```
Comment (jrobber) [inline Jared/lies.txt:2]: Math doesn't check out. You have bug.

Comment (jared-h-wood) [inline Jared/lies.txt:2]: It is correct because in lies.txt each statement must be false.

File: Brett/BrettFile.txt (hunk 1 above)
Comment (angolop) [inline Brett/BrettFile.txt:1]: that is so nice!!!

File: Jared/lies.txt (hunk 2 above)
Comment (stej01) [inline Jared/lies.txt:2]: You are bug.

**Pull request merged**